lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -208,6 +208,8 @@ def build_authorization_request(money, payment_method, options)
                 add_instalments_data(xml, options)
               end
               add_moto_flag(xml, options) if options.dig(:metadata, :manual_entry)
+              add_additional_3ds_data(xml, options) if options[:execute_threed] && options[:three_ds_version] && options[:three_ds_version].include?('2')
```

Comment:
This is probably ridiculous but theoretically `three_ds_version` could be `1.2`? So ideally this might be something like `options[:three_ds_version][0] == '2'`

Reply:
Also shouldn't this be `options[:three_d_secure][:version]`?

Reply:
So I believe we only send `options[:three_d_secure][:version]` when we pass the authenticated fields. For regular 3DS fields we send it in the options directly. Will update though so it checks it if starts with 2. Good catch!

Reply:
Oh I see, I'm conflating this with pass-through. Not seeing any remote tests, are we doing that external to AM?